fastq_filtrator.py

```diff
@@ -0,0 +1,96 @@
+
+def fun_gc_bounds(gc_bounds, GC_content):
+    left_border = 0.0
+    rigth_border = 100.0
+    if isinstance(gc_bounds, float) | isinstance(gc_bounds, int) == True:
+        rigth_border = float(gc_bounds)
+    else:
+        left_border = float(gc_bounds[0])
+        rigth_border = float(gc_bounds[1])
+    if left_border < GC_content < rigth_border:
+        return(True)
+    else:
+        return(False)
+
+
+def fun_length_bounds(length_bounds, counter):
+    left_border = 0.0
+    rigth_border = float(2**32)
+    if isinstance(length_bounds, float) | isinstance(length_bounds, int) == True:
+        rigth_border = float(length_bounds)
+    else:
+        left_border = float(length_bounds[0])
+        rigth_border = float(length_bounds[1])
+    if left_border < counter < rigth_border:
+        return(True)
+    else:
+        return(False)
+
+
+def fun_qual_thresh(qual_strok, quality_threshold = 0.0):
+    qual = 0
+    cou = 0
+    for k in qual_strok:
+        qual += ord(k)

```

Comment:
Ты забыла перевести в шкалу phred33
```suggestion
        qual += ord(k) - 33

```